src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add(number) {
+      return this.result + number;
+    },
+
+    subtract(number) {
+      return this.result - number;
+    },
+
+    multiply(number) {
+      return this.result * number;
+    },
+
+    divide(number) {
+      return this.result / number;
+    },
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics